digester/main.go

```diff
@@ -239,12 +246,26 @@ func main() {
 					}
 				}
 
+				text := ""
+				if message.SubType == "channel_join" {
+					text = "<em>Joined the channel</em>"
+				} else if message.SubType == "channel_archive" {
+					text = "<em>Archived the channel</em>"
+				} else if message.SubType == "channel_leave" {
+					text = "<em>Left the channel</em>"
+				} else if len(message.Blocks) > 0 {
+					text = parseMessage(message.Blocks)
+				}
+				if text == "" {
+					text = message.Text
+				}
+
```

Comment:
I guess a switch case would look more elegant here, maybe

Reply:
Handled in #38